src/TraceParserGen.Tests/TestBase.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.IO;
+using Xunit.Abstractions;
+
+namespace TraceParserGen.Tests
+{
+    /// <summary>
+    /// Base class for TraceParserGen tests
+    /// </summary>
+    public abstract class TestBase
+    {
+        protected static string OriginalInputDir = FindInputDir();
+        protected static string TestDataDir = "inputs";
+
+        protected TestBase(ITestOutputHelper output)
+        {
+            Output = output;
+            OutputDir = Path.Combine(Path.GetTempPath(), "TraceParserGen.Tests", Guid.NewGuid().ToString("N").Substring(0, 8));
+            
+            Directory.CreateDirectory(OutputDir);
+        }
+
+        protected ITestOutputHelper Output { get; }
+
+        protected string OutputDir { get; }
+
+        /// <summary>
+        /// Finds the input directory for test files
+        /// </summary>
+        private static string FindInputDir()
```

Comment:
This method is not needed - since you're operating from Environment.CurrentDirectory, the location of the inputs should be `Path.Combine(Environment.CurrentDirectory, "inputs")`.